discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -133,6 +133,11 @@ def set_ocs(self, ocs_enabled):
         ocs_operator = {"operators": [{"operator_type": "ocs", "enabled": ocs_enabled}]}
         self.api_client.update_cluster(self.id, ocs_operator)
 
+    def set_cnv(self, cnv_enabled):
```

Comment:
This is the new way: https://github.com/openshift/assisted-test-infra/pull/579/files

Reply:
Thanks, Will reuse

Reply:
@machacekondra rebased on top your changes and changed the code to reuse it for `CNV`